army.cpp

```diff
@@ -988,6 +988,22 @@ int Army::NumFront()
 	return (canfront + notfront - canbehind);
 }
 
+int Army::NumBehind() {
+	return NumAlive() - NumFront();
+}
+
+int Army::NumFrontHits() {
+	int totHits = 0;
+
+	for (int i = 0; i < canfront; i++) {
+		totHits += soldiers[i]->maxhits;
+	}
+	for (int i = canbehind; i < notfront; i++) {
+		totHits += soldiers[i]->maxhits;
+	}
+	return totHits;
+}
```

Comment:
just for sake of having it, mind adding NumBehindHits too?

Reply:
I can't guarantee that I will make it correct. I wanted to make it initially, but after testing the first version, I realized that I would need to invest much more time to understand what this class does. So I skipped this method.